assets/static/filters.test.js

```diff
@@ -1,7 +1,105 @@
+const $ = window.jQuery = require("jquery");
+require("./__mocks__/localStorageMock");
+
 test("filters addBadge()", () => {
-    // mock data attr with default filter
-    //document.body.innerHTML = "<div id='filter' data-default-filter='foo=bar'></div>";
-    window.jQuery = require("jquery");
     const filters = require("./filters");
     filters.addBadge("foo=bar");
 });
+
+test("default filter should be in history after setting filter to foo", () => {
+    localStorage.clear();
+    document.body.innerHTML =
+        "<div class='input-group filterbar'>" +
+        "  <div class='input-group-addon input-sm'>" +
+        "    <i class='fa fa-search' id='filter-icon'></i>" +
+        "  </div>" +
+        "  <input id='filter' type='text' value='foo' data-default-used='false' data-default-filter='default'>" +
+        "</div>" +
+        "<div id='historyMenu'></div>";
+
+    const templatesMock = require("./__mocks__/templatesMock");
+    document.body.innerHTML += templatesMock.loadTemplates();
+    const templates = require("./templates");
+    templates.init();
+
+    const autocomplete = require("./autocomplete");
+    const filters = require("./filters");
+
+    autocomplete.init();
+    filters.init();
+    filters.setFilters();
+    filters.renderHistory();
+
+    var elems = $("#historyMenu").find(".rawFilter");
+    // foo is active so should only get default
+    expect(elems.length).toBe(1);
```

Comment:
Would it be easier to use Jest's snapshot feature here (and again it the test below)?